src/Symfony/Component/Yaml/Tests/Command/LintCommandTest.php

```diff
@@ -142,6 +142,44 @@ public function testCustomTagsError()
         $this->assertSame(1, $ret, 'lint:yaml exits with code 1 in case of error');
     }
 
+    public function testLintWithExclude()
+    {
+        $tester = $this->createCommandTester();
+        $filename = $this->createFile('foo: bar');
+
+        $ret = $tester->execute(['filename' => $filename, '--exclude' => [$filename]], ['verbosity' => OutputInterface::VERBOSITY_VERBOSE, 'decorated' => false]);
+        $this->assertSame(0, $ret, 'lint:yaml exits with code 0 in case of success');
+        $this->assertStringContainsString('All 0 YAML files contain valid syntax.', trim($tester->getDisplay()));
+    }
+
+    public function testIncorrectLintConfig()
+    {
+        $this->expectException('RuntimeException');
+        $this->createCommandTester()->execute(['filename' => '.yamllint.yml'], ['decorated' => false]);
+    }
+
+    public function testCorrectLintConfig()
+    {
+        $tester = $this->createCommandTester();
+        $filename = $this->createFile('foo: bar');
+
+        $yaml = <<<YAML
+yaml-lint:
+  format: txt
+  includes:
+    - $filename
+  excludes: []
+YAML;
+
+        file_put_contents('.yamllint.yml', $yaml);
+
+        $ret = $tester->execute(['filename' => '.yamllint.yml'], ['decorated' => false]);
+
+        $this->assertSame(0, $ret, 'lint:yaml exits with code 0 in case of success');
+        $this->assertStringContainsString('All 1 YAML files contain valid syntax.', trim($tester->getDisplay()));
+        unlink('.yamllint.yml');
```

Comment:
We should use a tearDown() method to ensure the file is deleted after all tests, if one failed. Not sure how symfony handles tests like this

Reply:
Done, but not final. I will move the path to `sys_get_temp_dir()` with the `--config option`.

Reply:
There is now the `--config|-c` option without default value. I have also changed the tests, which will use the temp dir for the dummy lint config now.

BTW I have also added a shortcut `-e` for the exclude option.